src/syscheckd/create_db.c

```diff
@@ -318,9 +318,9 @@ static int read_file(const char *file_name, const char *linked_file, int dir_pos
                     free(esc_linked_file);
                     return -1;
                 }
-            } else if (OS_MD5_SHA1_SHA256_File(file_name, syscheck.prefilter_cmd, mf_sum, sf_sum, sf256_sum, OS_BINARY, syscheck.file_max_size) < 0)
+            } else if (OS_MD5_SHA1_SHA256_File(file_name, syscheck.prefilter_cmd, mf_sum, sf_sum, sf256_sum, OS_BINARY, syscheck.file_max_size * 1024 * 1024) < 0)
```

Comment:
I think you can create a function to convert from GB/MB/KB to bytes

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/4146/commits/2a91ed532de385e029597fd44fd239c79fca420b